src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.NonNetFx.cs

```diff
@@ -12,5 +14,52 @@ public static partial class PlatformDetection
         public static bool IsNetfx462OrNewer => false;
         public static bool IsNetfx470OrNewer => false;
         public static bool IsNetfx471OrNewer => false;
+
+
+        [DllImport("libc", ExactSpelling = true, CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
+        private static extern IntPtr gnu_get_libc_release();
+
+        [DllImport("libc", ExactSpelling = true, CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
+        private static extern IntPtr gnu_get_libc_version();
+
+        /// <summary>
+        /// If gnulibc is available, returns the release, such as "stable".
+        /// If the libc is musl, currently returns empty string.
+        /// Otherwise returns "Glibc not found".
+        /// </summary>
+        public static string LibcRelease
+        {
+            get
+            {
+                try
+                {
+                    return Marshal.PtrToStringUTF8(gnu_get_libc_release());
```

Comment:
Do we not build this lib on netfx?

Reply:
It's implemented in corelib, so no. We don't have it in an OOB.

Reply:
Ah, I thought we'd ported it. I guess not yet.